### DISCUSSION TITLE
DialogHost and XAML Binding Error

### DISCUSSION THREAD

**Discussion by Frillaut** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3204] [Q&A] [answered]
Hi!
This simple thing leads to an error in the XAML binding... Can't DialogHost be inserted into the root Window element?

```xaml
<Window ...>
    <materialDesign:DialogHost/>
</Window>
```

XAML Binding Error
Data context: null
Target: VisualBrush.Visual
Target Type: Visual
Description: The control FrameworkElement or FrameworkContentElement for the target element cannot be found.

**Comment by Keboo** [comment, marked as answer]
I believe this is a duplicate of #3228
